roles/setup/tasks/preflight.yml

```diff
@@ -1,5 +1,5 @@
 ---
 - name: Check local binary file is set
   ansible.builtin.assert:
-    that: receptor_local_bin_file | length
+    that: receptor_local_bin_file | length > 0
```

Comment:
Shouldn't we define a default here as a part of the that clause?

Reply:
Good suggestion. I've submitted a new commit to address that in a similar fashion like `tls_local.yml`.